### PR TITLE
parameter for partitions

### PR DESCRIPTION
**Pull request by devcateu**

### CODE REVIEW

File: core/src/main/scala/akka/persistence/postgres/config/AkkaPersistenceConfig.scala [hunk 1]
```diff
@@ -85,6 +92,7 @@ class SnapshotPluginConfig(config: Config) {
 // aggregations
 
 class JournalConfig(config: Config) {
+  val partition = new PartitionedJournal(config)
```
Comment (mkubala) [inline core/src/main/scala/akka/persistence/postgres/config/AkkaPersistenceConfig.scala:95]: `partitionsConfiguration`?

File: core/src/main/scala/akka/persistence/postgres/config/AkkaPersistenceConfig.scala [hunk 2]
```diff
@@ -57,8 +57,15 @@ class SnapshotTableConfiguration(config: Config) {
   override def toString: String = s"SnapshotTableConfiguration($tableName,$schemaName,$columnNames)"
 }
 
+class PartitionedJournal(config: Config) {
```
Comment (mkubala) [inline core/src/main/scala/akka/persistence/postgres/config/AkkaPersistenceConfig.scala:60]: `JournalPartitionsConfig`?

File: core/src/main/resources/reference.conf [hunk 3]
```diff
@@ -109,6 +109,11 @@ jdbc-journal {
 
   tables {
     journal {
+      # The size of single partition and prefix of partition when we choose to use partitioned journal.
+      partitioned {
```
Comment (mkubala) [inline core/src/main/resources/reference.conf:113]: We already have a config value called `partitioned` on the root level of the journal configuration. To make it less ambiguous and readable more like prose I'd suggest rename this key to `partitions`.

**Comment by mkubala** [comment]
Just naming stuff.

File: core/src/main/scala/akka/persistence/postgres/config/AkkaPersistenceConfig.scala (hunk 1 above)
Comment (devcateu) [reply core/src/main/scala/akka/persistence/postgres/config/AkkaPersistenceConfig.scala]: changed

File: core/src/main/scala/akka/persistence/postgres/config/AkkaPersistenceConfig.scala (hunk 2 above)
Comment (devcateu) [reply core/src/main/scala/akka/persistence/postgres/config/AkkaPersistenceConfig.scala]: changed

File: core/src/main/resources/reference.conf (hunk 3 above)
Comment (devcateu) [reply core/src/main/resources/reference.conf]: changed

File: core/src/main/scala/akka/persistence/postgres/config/AkkaPersistenceConfig.scala [hunk 4]
```diff
@@ -49,16 +49,23 @@ class SnapshotTableColumnNames(config: Config) {
   override def toString: String = s"SnapshotTableColumnNames($persistenceId,$sequenceNumber,$created,$snapshot)"
 }
 
-class SnapshotTableConfiguration(config: Config) {
+class SnapshotTableConfig(config: Config) {
```
Comment (mkubala) [inline core/src/main/scala/akka/persistence/postgres/config/AkkaPersistenceConfig.scala:52]: No need to rename old configuration classes. I just suggested to keep the name for the new one consistent with the existing pattern.

Comment (devcateu) [reply core/src/main/scala/akka/persistence/postgres/config/AkkaPersistenceConfig.scala:52]: Why do not keep consistency across all classes?

Comment (mkubala) [reply core/src/main/scala/akka/persistence/postgres/config/AkkaPersistenceConfig.scala:52]: Let's do it in a separate PR. As long as two devs work on the same part of the codebase, refactoring should be postponed.

**Pull request merged**